lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -879,81 +891,48 @@ class SemanticsObject {
       effectiveTransformIsIdentity = false;
     }
 
-    if (!effectiveTransformIsIdentity || isMacOrIOS) {
-      if (effectiveTransformIsIdentity) {
-        effectiveTransform = Matrix4.identity();
-      }
-      if (isDesktop) {
-        element.style
-          ..transformOrigin = '0 0 0'
-          ..transform = (effectiveTransformIsIdentity ? 'translate(0px 0px 0px)'
-              : matrix4ToCssTransform(effectiveTransform));
-      } else {
-        // Mobile screen readers observed to have errors while calculating the
-        // semantics focus borders if css `transform` properties are used.
-        // See: https://github.com/flutter/flutter/issues/68225
-        // Therefore we are calculating a bounding rectangle for the
-        // effective transform and use that rectangle to set TLWH css style
-        // properties.
-        // Note: Identity matrix is not using this code path.
-        final ui.Rect rect =
-            computeBoundingRectangleFromMatrix(effectiveTransform, _rect!);
-        element.style
-          ..top = '${rect.top}px'
-          ..left = '${rect.left}px'
-          ..width = '${rect.width}px'
-          ..height = '${rect.height}px';
-      }
+    if (!effectiveTransformIsIdentity) {
```

Comment:
have you verified https://github.com/flutter/flutter/issues/68225 is fine using this version?

Reply:
Yes, this PR replaces the previous fix with `updateSemanticsScreenProperties` (see `dom_renderer.dart`). I tested both on Android and iOS (desktop didn't change as it already used `transform` due to `if (isDesktop)`).